tests/Fixtures/Entity/Product.php

```diff
@@ -0,0 +1,52 @@
+<?php
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DoctrineORMAdminBundle\Tests\Fixtures\Entity;
+
+class Product
```

Comment:
Could we make this class final?

Reply:
Doctrine does not allow use a final entities, but in tests we can make it as final.